impress-org/givewp
# New: Add promotion of free add-on for free users

## Description

This adds promotional offerings for a free add-on. This includes:
- A Free Add-On tab to the add-ons page
- A modal that pops up upon updating GiveWP (and the next two versions if dismissed)
- A green button that links to the add-on page on all Give Settings pages

## Visuals



## Testing Instructions

1. Use the different forms to subscribe
2. Ask someone with Active Campaign access to confirm new user and tags

## Pre-review Checklist
